packages/design-system-react/package.json

```diff
@@ -51,10 +51,7 @@
   },
   "dependencies": {
     "@metamask/jazzicon": "^2.0.0",
-    "@metamask/utils": "^11.4.0",
     "@radix-ui/react-slot": "^1.1.0",
-    "@solana/addresses": "^2.1.0",
-    "bitcoin-address-validation": "^3.0.0",
```

Comment:
These dependencies have been moved to peerDependencies because they are used by the extension and portfolio outside of the design system’s UI components, so moving them to peerDependencies allows the consuming app to specify the version, reducing the overall bundle size and avoiding duplicate dependencies. We are keeping @metamask/jazzicon as a regular dependency because it should only be used internally by AccountAvatar within design-system-react, not directly by consuming applications.